src/main/java/screens/chat_screen/ChatView.java

```diff
@@ -0,0 +1,231 @@
+package screens.chat_screen;
+
+
+import use_cases.chat_initiation_use_case.ChatInteractor;
+import use_cases.chat_initiation_use_case.CheckUsername_Interactor;
+
+import javax.swing.*;
+import javax.swing.border.EmptyBorder;
+
+
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+
+/**
+ * ChatView is our UI for Private chat.It Containe a chatframe .At the top of the frame
+ * there is a text file in which we type userB's username, and  chat frame will change into
+ * "userB'S username." AT the bottom of the frame, there is another text field to type a message and
+ * send button to send the message
+ *
+ * In the middle of the frame there i sJpanel and Jlabels for the converstaion history.
+ *
+ */
+
+
+
+class ChatView extends JFrame implements  ActionListener{
+    //use two Interactors .
+    private ChatInteractor chatInteractor;
+    private CheckUsername_Interactor checkusername_interactor;
+
+    //Use Jframes, butttons, labels ,textfileds, Jpannels,JMenuBar for UI.
+    final JFrame frame ;
+    private JButton addbutton;
+    final JButton sendbutton;
+    final JLabel l;
+    final JLabel label;
+    final JTextField usernametextfield;
+    final JTextField messagetextfield;
+    final JMenuBar menubar;
+    final JPanel panel;
+
+    final JPanel conversationHistoryPanel;
+
+    private JPanel messagePanel;
+    private JLabel testMessageHeader;
+    private JLabel testMessage;
+
+    //isNewchat check we already have a chat with a user
+    private boolean isNewchat;
+
+
+
+
+    //this is constructor
+    public ChatView( boolean isNewchat){
+
```

Comment:
I think we were talking about this earlier but in the appscreen display when the add new chat button is clicked it gives the username to ChatView. I'm not sure if you need this for ChatView (or conversation history later), so feel free to let me know if I should remove that from my part